### ISSUE TITLE
set up cb-csv demo

### ISSUE THREAD

**Issue by evanwill** [CollectionBuilder/collectionbuilder-csv#41]
requires github action, which I don't really want in the main branch of the repository (because it causes weird issues from people copying it). So probably need to set up a separate demo project repository?

**Comment by maehr** [comment]
I would try to keep it in one repo. I have really good experiences with GH actions that deploy to a separate branch (probably also as gh-page). A quick fix could be Cloudflare Pages, Vercel et. al.

**Comment by evanwill** [comment]
@maehr as far as I know, gh action workflows have to be in the default branch to run (even if they are using other branches)? That has tended to cause issues and confusion when people make a copy with "use this template", since they end up with an action running in their own repo that results in a broken site or build errors (because it isn't configured correctly for their repo). 
The third party build option from a demo-site branch might be best option. We have mostly used Render, but could check out the other services too.

**Comment by maehr** [comment]
I agree with you. The third party option is the cleanest and simplest solution. Alternatively, you could create a GH Action`.github/workflows/demosite.yaml` that only runs on the demosite-branch

```yaml
on:
  push:
    branches:
      - demosite
 
(...)
```

The YAML file is still visible, but it does not trigger Actions when you use the repo as a template.
